web-ui/src/main/resources/catalog/locales/ru-editor.json

```diff
@@ -300,7 +300,7 @@
     "suggestions": "Suggestions",
     "removeThumbnail": "Remove thumbnail",
     "thumbnailMaker": "Create thumbnail",
-    "thumbnailMaker-help": "All WMS layers registered in the metadata record are added to the map. Choose a layout (Thumbnail recommended), a scale, zoom to the area of interest to generate the thumbnail.",
+    "thumbnailMaker-help": "All WMS layers registered in the metadata record are added to the map. Choose a layout, a scale, zoom to the area of interest to generate the thumbnail.",
```

Comment:
I believe this translation is handled by transifex and should not be included in the pull request.

Reply:
I see. I will restore it.